docs/sdks/server-sdks/node.md

```diff
@@ -330,30 +330,211 @@ To read more about different flag types, see the page on [Flag Variations](/feat
+|----------------------------------|--------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| `experiment` (string)            | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17"             |
+| `subject` (string)               | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                            |
+| `variation` (string)             | The experiment variation the subject was assigned to                                                                     | "control"                                       |
+| `timestamp` (string)             | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z                        |
+| `subjectAttributes` (Attributes) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`                           |
+| `featureFlag` (string)           | An Eppo feature flag key                                                                                                 | "recommendation-algo"                           |
+| `allocation` (string)            | An Eppo allocation key                                                                                                   | "allocation-17"                                 |
+| `holdout` (string)               | An Eppo holdout group key                                                                                                | "q1-holdout"                                    |
+| `holdoutVariation` (string)      | An Eppo holdout variation if experiment is eligible for analysis key                                                     | "status_quo", "all_shipped_variations", or null |
+
+:::note
+The `Attributes` type represents a mapping of an attribute name to its value, which could be a string, number or boolean (`Record<string, string | number | boolean>`).
+:::
 
 :::note
 More details about logging and examples (with Segment, Rudderstack, mParticle, Snowplow, Amplitude) can be found in the [event logging](/sdks/event-logging/) page.
 :::
+
+## Usage with Contextual Multi-Armed Bandits
+
+Eppo also supports contextual multi-armed bandits. You can read more about them in the [high-level documentation](../../../contextual-bandits).
+Bandit flag configuration--including setting up the flag key, status quo variation, bandit variation, and targeting rules--are configured within
+the Eppo application. However, available actions are supplied to the SDK in the code when querying the bandit.
```

Comment:
@hhargreaveseppo added in this paragraph based on your feedback. It links to our detailed documentation on bandits, and also mentions how the setup occurs in the product while actions are supplied in the code.